src/platform/functions/index.js

```diff
@@ -83,19 +83,42 @@ const scrapeCityEvents = async (city) => {
           parsedImgUrl = imgUrl.getAttribute('data-src');
         }
 
+        let lat = 0;
+        let lng = 0;
+        if (eventLoc) {
+          try {
+            const response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${eventLoc.innerText}&key=AIzaSyCFm3jPZv8BgY7sXh5QS8X6WNpApmfD8OQ`);
+            const geoData = await response.json();
+            lat = await geoData.results[0].geometry.location.lat;
+            lng = await geoData.results[0].geometry.location.lng;
+          }
+          // if (geoData.results.length > 0) {
+            
+          // }
+          // else {
+          //   lat = NaN // Invalid location
+          //   lng = NaN // Invalid location
+          // }
+        // } else {          // Else If no event location found then
+          catch {
+            lat = NaN // Invalid location
+            lng = NaN // Invalid location
```

Comment:
```suggestion
            lat = "" // Invalid location
            lng = "" // Invalid location
```
hmm, NaN will probably mess things up on firestore if we deploy it

Reply:
yeah I checked on local emulator it makes it null, empty string is better